portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -262,9 +262,39 @@ definition {
 		var key_currentAssignee = "${currentAssignee}";
 		var key_currentDateDay = DateUtil.getCurrentDate();
```

Comment:
Hi Melody,
Thank you for making these changes. I noticed what was going on, and it seems that we have a method to format dates on DateUtil. So, I changed only the value of key_currentDateDay and it was all good (+ adding key_ to the other variables).

var key_currentDateDay = DateUtil.getFormattedDate("0", "dd");